.github/workflows/pcw.yml

```diff
@@ -11,11 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: ['3.8', '3.10']
     steps:
       - uses: actions/checkout@v2
       - name: Setup Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
```

Comment:
does it mean that we were using python2 here or what ?

Reply:
ah it is just **package** version , right ?

Reply:
right , its action version.